CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jespino
```

Comment:
@jasonblais I added a `CODEOWNERS` to ensure that every PR gets reviewed by @jespino. Other plugins use a similar file.

Reply:
Works for me :+1: